code/modules/mining/minebot.dm

```diff
@@ -36,20 +36,18 @@
 	del_on_death = TRUE
 	var/mode = MINEDRONE_COLLECT
 	var/light_on = 0
-
-	var/datum/action/innate/minedrone/toggle_light/toggle_light_action
-	var/datum/action/innate/minedrone/toggle_mode/toggle_mode_action
-	var/datum/action/innate/minedrone/dump_ore/dump_ore_action
 	var/obj/item/gun/energy/kinetic_accelerator/minebot/stored_gun
 
 /mob/living/simple_animal/hostile/mining_drone/Initialize()
 	. = ..()
 	stored_gun = new(src)
-	toggle_light_action = new()
+	var/datum/action/innate/minedrone/toggle_light/toggle_light_action = new()
```

Comment:
these really should be qdel'd with the drone

Reply:
done in the commit below